### PR TITLE
Adding Skale chain config

### PR DESCRIPTION
**Pull request by AzizBardi**
Skale chain Added with all info

### CODE REVIEW

**Comment by portuu3** [comment]
@AzizBardi is it ready to review?

**Comment by AzizBardi** [comment]
> @AzizBardi is it ready to review?

Yes !

File: packages/apps/escrow-dashboard/src/constants/index.ts [hunk 1]
```diff
@@ -58,6 +81,7 @@ export const SUPPORTED_CHAIN_IDS = [
   ChainId.MOONBASE_ALPHA,
   ChainId.AVALANCHE_TESTNET,
   ChainId.AVALANCHE,
+  ChainId.SKALE,
```
Comment (portuu3) [inline packages/apps/escrow-dashboard/src/constants/index.ts:84]: please remove it from here until we have a subgraph maintainer

**Review by portuu3 — CHANGES_REQUESTED** [review]
fix lint errors

File: packages/apps/escrow-dashboard/src/constants/index.ts (hunk 1 above)
Comment (AzizBardi) [reply packages/apps/escrow-dashboard/src/constants/index.ts:84]: done !

**Review by portuu3 — APPROVED** [review]

**Comment by portuu3** [comment]
@AzizBardi what are those contract error calls ?
image

**Pull request merged**